.gitignore

```diff
@@ -34,3 +34,4 @@ queue_listeners/config/env.yml
 !.elasticbeanstalk/*.global.yml
 node_modules
 public/system
+config/unicorn.rb
```

Comment:
why is this gitignored? can you add a template? does this mean that unicorn is ready to rock?